new/LIT5885HymnMary.xml

```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5885HymnMary" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኦእግዝእትየ፡ ማርያም፡ ጽዋዓ፡ ልቡና፡ መለኮታዊ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾO-ʾǝgzǝʾǝtǝya Māryām ṣǝwāʿā lǝbbunā malakotāwi</title>
```

Comment:
If the meaning, as it seems to me, is "O Our lady Mary, chalice of divine intelligence", then the word is ጽዋዐ፡ (status constructus from ጽዋዕ፡). The word is spelled _ṣǝwwāʿ_ in Leslau, _Comp. Dict._ 566a, with double _w_. So ጽዋዐ፡ and ṣǝwwāʿa.

Reply:
Perfect, thank you! And why not add the English title...